.github/workflows/build-deploy.yml

```diff
@@ -26,22 +26,37 @@ jobs:
   #         version: v1.60.3
   #         args: --timeout 600s --verbose ./...
 
-  test:
+  codacy-analysis:
```

Comment:
I would keep the previous naming `test`, as the codacy part is just reporting on the tests that were done

Reply:
done